launchers/common/base/build.gradle.kts

```diff
@@ -22,6 +22,7 @@ dependencies {
     // sovity Extensions Package
     api(project(":extensions:sovity-edc-extensions-package"))
     api(project(":extensions:postgres-flyway"))
+    api(project(":extensions:transfer-process-status-checker"))
```

Comment:
Can you elloborate on why this extensions has not been added to the `sovity-edc-extensions-package`?